.github/workflows/go-check.yaml

```diff
@@ -32,6 +32,23 @@ jobs:
         with:
           version: v1.27
 
+  precheck:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.14.6
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          # hard-code the path instead of using ${{ github.repository }} to make sure it works for forked repo as well
+          path: src/github.com/cilium/cilium
+      - name: Go code prechecks
```

Comment:
This is just a question :). 

I noticed that precheck target will run check-k8s-code-gen.sh based on env SKIP_K8S_CODE_GEN_CHECK, what do you think about removing generate-k8s-api step below, and run it here as well ?

Reply:
True, but I'd rather keep them as separate checks. I think that way it's easier for contributors to spot issues.